compiler/ast/ast.nim

```diff
@@ -10,14 +10,25 @@
 ## abstract syntax tree + symbol table
 
 import
-  lineinfos, hashes, options, ropes, idents, int128, tables
-
-from strutils import toLowerAscii
-
-import ./ast_types
-export ast_types
-
-export int128
+  compiler/ast/[
+    lineinfos, # Positional information
+    idents, # Ast identifiers
```

Comment:
Sorry I fell really compelled to be nitpicky here -- just because it's going to end up everywhere.

Can we align the comments so all the starting # line up in instance where we gave lists like this?